web/modules/custom/ecc_d10/README.md

```diff
@@ -1 +1,2 @@
-# Temporary modules for Drupal 10 upgrade&#10&#10Delete this folder after the Drupal 10 upgrade.
\ No newline at end of file
+# Temporary modules for Drupal 10 upgrade
```

Comment:
These modules will be deleted as part of LP-59 (Tidy up essex-gov-uk-drupal repository)